content/learn/fundamental-wellbeing.md

```diff
@@ -1,73 +1,96 @@
+ 
+ While there are many terms for spiritual enlightenment, here we refer to the term Fundamental Wellbeing. In fact, this page is dedicated to Dr. Jeffery Martin's work, the man who coined the term and has spent over a decade researching how people reach Fundamental Wellbeing. 
+ 
+ By the end of this page you should have a solid understanding of Dr. Martin's work, the concepts of Fundamental Wellbeing and how to explore it further.
 
-Fundamental Wellbeing is a termed coined (or made popular) by [Jeffery Martin](https://lifeitself.org/notes/jeffery-martin). In more academic settings he tends to refer to it as Persistent Non-Symbolic Experience (Martin credits the "non-symbolic" terminology to [abc/@cook-greuter](https://lifeitself.org/abc/@cook-greuter)).
+## Interviews with Dr. Jeffery Martin
 
-The terminology is a secular (and hence friendlier) and standardized label for [state](https://lifeitself.org/state) stages. Martin's map of the transcendent section of state space is the most detailed and worked out of any we know of. It greatly extends [Wilber](https://lifeitself.org/Wilber)'s map and those of classic wisdom traditions.
+Before we explore Fundamental Wellbeing throughout this page, begin by watching one of the most extensive interviews with Dr. Jeffery Martin on the web. 
 
-The most recent version of Martin's map is from his keynote at the Society for Consciousness Studies 2021 Conference: [The Martin Matrix, a Comprehensive Map of Persistent Non-Symbolic Experience (Enlightenment/Nondual)](https://www.youtube.com/watch?v=aCfeamM07dk) (YouTube Dec 22 2021).
+Hosted by Life Itself Co-founder, Rufus Pollock, these interviews explore the many facets of Fundamental Wellbeing, Dr. Martins research and how this has informed his work developing tools and protocols for reaching Fundamental Wellbeing.
 
-> The Martin Matrix, a Comprehensive Map of Persistent Non-Symbolic Experience In the spirit of previous milestones such as the "Wilber-Combs Matrix", this talk presents a new comprehensive model covering different types and depths of Persistent Non-Symbolic Experience (PNSE; i.e.: enlightenment, nonduality, the peace that passeth understanding, unity consciousness, etc.). Publications and presentations over the previous 15 years of research have extensively detailed the “types” axis of the matrix. This was often confusing to the interested public, because most previous models by religious, spiritual, and philosophical traditions and teachers primarily refer to experiences in the “depth” axis of the matrix. This is the first academic presentation covering the “depth” axis, and will also explore how various depths of PNSE change across its types, which degree of depth is most resonant with a given type, and so on.
+### Part 1: Researching awakened people
```

Comment:
Part of the useful work here would be to break up the interviews into small segments (but hey that's later!)